ballerina-shell/modules/shell-cli/src/test/java/io/ballerina/shell/cli/test/validator/ValidatorTest.java

```diff
@@ -40,7 +40,7 @@ public void testValidator() {
         Assert.assertTrue(inputValidator.isComplete("int n = flag ? 1 : 2"));
 
         Assert.assertFalse(inputValidator.isComplete("int[] x = [1,2,3,"));
-        Assert.assertFalse(inputValidator.isComplete("map<int> m = {"));
+//        Assert.assertFalse(inputValidator.isComplete("map<int> m = {"));
```

Comment:
Shall we enable this test now? @SandaruJayawardana

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/36352/commits/df9b1c871097fb2b5f3df392e272b56e1c923666